kola-denylist.yaml

```
@@ -99,3 +99,9 @@
 
 - pattern: ext.config.shared.multipath.resilient
   tracker: https://github.com/coreos/fedora-coreos-tracker/issues/1937
+
+- pattern: ext.config.security.lockdown
```

```suggestion
- pattern: ext.config.security.lockdown
  tracker: https://github.com/openshift/os/issues/1237
```

All the other Secureboot related entries here point to this issue